db/connections.go

```diff
@@ -98,6 +98,11 @@ func PersistConnectionFromCRD(ctx context.Context, obj *v1.Connection) error {
 		dbObj.Password = obj.Spec.GitHub.PersonalAccessToken.String()
 	}
 
+	if obj.Spec.GitLab != nil {
+		dbObj.Type = models.ConnectionTypeGitlab
```

Comment:
I think we need an optional URL here as well - Github and Azure devops should also have

Reply:
@moshloop why do we need urls for GitHub connection?